security/sensorhub-security-oauth/build.gradle

```diff
@@ -3,15 +3,22 @@ ext.details = 'Provides federated authentication through OAuth2 providers such a
 version = '0.9.0'
 
 dependencies {
-  implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
-  embeddedImpl 'org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.1'
+  dependencies {
+    implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
+    embeddedImpl('org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.1') {
+      exclude group: 'org.slf4j', module: 'slf4j-api'
+    }
+  }
 }
 
 // add info to OSGi manifest
 osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
     attributes('Bundle-Activator': 'org.sensorhub.impl.security.oauth.Activator')
+    attributes('Import-Package': '''
```

Comment:
I'm not sure this is needed since we exclude these packages by default in common.gradle

Reply:
Does not install w/o these lines:

```sh
org.osgi.framework.BundleException: Unable to resolve org.sensorhub.sensorhub-security-oauth [53](R 53.0): missing requirement [org.sensorhub.sensorhub-security-oauth [53](R 53.0)] osgi.wiring.package; (osgi.wiring.package=javax.security.auth) Unresolved requirements: [[org.sensorhub.sensorhub-security-oauth [53](R 53.0)] osgi.wiring.package; (osgi.wiring.package=javax.security.auth)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4398)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2308)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:1007)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:993)
	at org.sensorhub.impl.osgi.OsgiLauncher.installBundles(OsgiLauncher.java:161)
	at org.sensorhub.impl.osgi.OsgiLauncher.main(OsgiLauncher.java:267)
```